Write ETAS rpts local, cp .gz to persistent storage

jsteverman opened the pull request:
Adds a step to the generation of the ETAS overlap reports. It now writes to a local directory by default. Once completed, the gz files are copied to the /htprep and dropbox locations.

jsteverman commented:
I don't get it. 
`cronJob.emptyVolumeMount('tmp','/tmp',volumeMixin=volume.emptyDir.withMedium('Memory'))` 
seems like writing to /usr/src/app/tmp with extra steps.

aelkiss commented:
Discussed via slack; the `emptyVolume` mount is necessary to use RAM rather than the k8s worker node local disk.

jsteverman commented:
I added `local_report_path: "/tmp_reports"` to the holdings configmap. 

And added the tmp_reports volume in tanka:
https://github.com/hathitrust/ht_tanka/pull/35

aelkiss commented:
:+1: :ship:
